output/createMaxCSV.sh

```diff
@@ -19,115 +19,239 @@
+   SCENARIOLOG=${properties['monitoring.logging.file.scenariolog']}
 fi
 # this script can be called with just one command line option: the
 # full path to the run.properties file
 echo "Loading properties."
-# get loadProperties function
-SCRIPTDIR=`sed -n 's/[ ^]*$//;s/path.scriptdir\s*:\s*//p' $RUNPROPERTIES`
-source $SCRIPTDIR/properties.sh
 # load run.properties file into associative array
+source $SCRIPTDIR/properties.sh
 loadProperties $RUNPROPERTIES
 echo "Finished loading properties."
 # now set variables that would otherwise be set by command line arguments
-CONFIG=${properties['config.file']}
-CYCLEDIR=${properties['path.advisdir']}
 CYCLE=${properties['advisory']}
-HPCENV=${properties['hpc.hpcenv']}
 SCENARIO=${properties['scenario']}
-CSDATE=${properties['adcirc.time.coldstartdate']}
-HSTIME=${properties['InitialHotStartTime']}
 GRIDFILE=${properties['adcirc.file.input.gridfile']}
-OUTPUTDIR=${properties['path.outputdir']}
-SYSLOG=${properties['monitoring.logging.file.syslog']}
-SSHKEY=${properties['post.file.sshkey']}
-HPCENVSHORT=${properties['hpc.hpcenvshort']}
-HPCENV=${properties['hpc.hpcenv']}
+GRIDNAME=${properties['adcirc.gridname']}
+BACKGROUNDMET=${properties['forcing.backgroundmet']}
 TROPICALCYCLONE=${properties['forcing.tropicalcyclone']}
 if [[ $TROPICALCYCLONE != "off" ]]; then
    STORM=${properties['forcing.tropicalcyclone.stormnumber']}
    YEAR=${properties['forcing.tropicalcyclone.year']}
 else
    STORM="null"
    YEAR=${CYCLE:0:4}
-fi      
-#
-SCENARIODIR=${CYCLEDIR}/${SCENARIO}       # shorthand
-CYCLELOG=${properties['monitoring.logging.file.cyclelog']}
-SCENARIOLOG=${properties['monitoring.logging.file.scenariolog']}
-source ${SCRIPTDIR}/monitoring/logging.sh
-source ${SCRIPTDIR}/platforms.sh
-# dispatch environment (using the functions in platforms.sh)
-env_dispatch ${HPCENVSHORT}
-THIS=output/createMaxCSV.sh
-allMessage "$SCENARIO: $THIS: Starting post processing."
-scenarioMessage "$THIS: SCENARIO=$SCENARIO ; SCENARIODIR=$SCENARIODIR"
-cd ${SCENARIODIR} 2>&1 > errmsg || warn "cycle $CYCLE: $SCENARIO: $THIS: Could not change directory to $SCENARIODIR: `cat $errmsg`"
+fi
 #
 #-----------------------------------------------------------------------
-#     C R E A T E   M A X   C S V  
+#     C R E A T E   M A X   C S V
 #------------------------------------------------------------------------
 # form the csv file name, e.g.: jose2017adv44HSOFSnhcConsensusMax.csv
-STORMNAMELC=nam
+
 if [[ $TROPICALCYCLONE != off ]]; then
    STORMNAME=${properties['forcing.tropicalcyclone.stormname']}
    # make the storm name lower case
-   STORMNAMELC=`echo $STORMNAME | tr '[:upper:]' '[:lower:]'` 2>> ${SYSLOG}
+   STORMNAMELC=$(echo $STORMNAME | tr '[:upper:]' '[:lower:]') 2>> ${SYSLOG}
+   csvFileName="${STORMNAMELC}${YEAR}adv${CYCLE}${GRIDNAME}${SCENARIO}Max.csv"
+else
+   if [[ $BACKGROUNDMET == "on" || $BACKGROUNDMET == "nam" ]]; then
+      STORMNAMELC=nam
+   fi
```

Comment:
`elif`? Or better yet a maybe a `case` statement? I am just thinking of scalability in case this has to grow.

Reply:
Done